data/bookkeeping/block.go

```diff
@@ -131,6 +131,13 @@ type (
 		ParticipationUpdates
 	}
 
+	// TxnRoot represents the root of the merkle tree generated from the transaction in this block.
+	TxnRoot struct {
```

Comment:
wonder if we can somehow write a test to ensure the old serialized and the new formats match

Reply:
I can use an instance of old serialized `BlockHeader` hardcoded in some test and make sure it can be de-serialized correctly into the updated struct.